extension-ui/oops.js

```diff
@@ -1,5 +1,33 @@
-document.addEventListener("DOMContentLoaded", function() {
+
+import React, { useEffect } from 'react';
+import ReactDOM from 'react-dom';
+import GenericButton from '../client-side/src/stories/Button/GenericButton'; 
```

Comment:
You can't import form client-side folder directly, it's separated service.
you should import it from the storybook that you've bundled, install it as node module in the extension, and use it.